src/sssom/util.py

```diff
@@ -194,6 +213,54 @@ def remove_mappings(self, msdf: "MappingSetDataFrame") -> None:
         self.clean_prefix_map()
 
 
+def _standardize_curie_or_iri(curie_or_iri: str, *, converter: Converter) -> str:
+    """Standardize a CURIE or IRI, returning the original if not possible.
+
+    :param curie_or_iri: Either a string representing a CURIE or an IRI
+    :returns:
+        - If the string represents an IRI, tries to standardize it. If not possible, returns the original value
+        - If the string represents a CURIE, tries to standardize it. If not possible, returns the original value
+        - Otherwise, return the original value
+    """
+    if is_iri(curie_or_iri):
+        return converter.standardize_uri(curie_or_iri) or curie_or_iri
+    if is_curie(curie_or_iri):
+        return converter.standardize_curie(curie_or_iri) or curie_or_iri
+    return curie_or_iri
+
+
+def _standardize_metadata(converter, metadata) -> None:
```

Comment:
this method does much more than standardising. Its more like a repair! For example, while you cant use a | separate in a URL, I _think_ you can use it in an IRI. But well, mooooot, I guess if you start promoting IRI grammar rules we will get into hells kitchen. So yea, probably ok.

Reply:
(You can ignore this comment, nothing actionable)